src/themes/components/tabs.ts

```diff
@@ -1,11 +1,14 @@
+import { opacity } from '../shared/opacity';
+import { borders } from '../shared/borders';
+
 const baseStyleTab = {
   textStyle: 'body',
   transitionProperty: 'common',
   transitionDuration: 'normal',
-  borderBottom: '1.5px solid',
+  borderBottom: borders['1px'],
```

Comment:
Sergiu confirmed 1px 👍

Reply:
❤️

Reply:
For the borders you don't need to reference it with an object.

Chakra has the key for the borders but no for opacity. The reason why we need it only for opacity